test/data/estimate-self-assessment-penalties-questions-and-responses.yml

```diff
@@ -1,30 +1,29 @@
 ---
 :which_year?:
-- 2011-12
```

Comment:
@ikennaokpala it is a bit difficult to follow this PR. There are many artefacts changing.
For example, in this file we are removing 2011-2012 and also updating other values. It is not clear to me why.
Tests are not also passing in each commit. Can you think in a way to make it easier to review?